### DISCUSSION TITLE
Color of elements not changing as per wallpaper, (Or how to do so?)

### DISCUSSION THREAD

**Discussion by MeditatedBison** [end-4/dots-hyprland#414] [Q&A] [answered]
i think there were some conflicting dependency errors while installing these dotfiles. 
I changed the wallpaper through swww, but the color of elements on top bar are not changing. I even chose the color picker, and picked some color, but there was no change. 

please tell a solution or way?

Thanks in advance!

**Comment by end-4** [comment]
> i think there were some conflicting dependency errors while installing these dotfiles

attempt the installation again and show logs

**Reply by MeditatedBison** [reply]
The following is the first point where error comes:
![20240415_12h35m05s_grim](https://github.com/end-4/dots-hyprland/assets/95769851/d147f453-920e-44a2-8aee-f1866e6f82c7)
I ran the command separately in a different terminal, it gave the same error, i have no idea how to solve this conflicting dependency.
So i pressed i and ignored this 

Then the installation was finished in some time.

**Reply by MeditatedBison** [reply]
I ran the command, Without the `--noconfirm` again. And in yay when it asks built is already there do you need to cleanbuild? I chose `All` . instead of the default `No` that the script does.

And when it asked to remove the conflicting package, I pressed `Y` , but still it gave a error

![20240415_12h56m02s_grim](https://github.com/end-4/dots-hyprland/assets/95769851/2b2df956-a26c-486f-ad52-38efca9e914a)

**Reply by end-4** [reply, marked as answer]
uninstall `hyprpicker-debug` (non -git) maybe

**Reply by MeditatedBison** [reply]
Thank you so much, I ran the script again after doing that, and the setup is very beautiful!

**Comment by MeditatedBison** [comment]
I pressed Ctrl + Win + T and selected a image, But then there is a error!!

The following picture shows the error at top, and the colors.conf file also
![20240415_12h45m49s_grim](https://github.com/end-4/dots-hyprland/assets/95769851/082749f8-dc1d-4533-8e21-2e5de4b7284c)

**Reply by gnussbaum67** [reply]
I'm seeing the same error.
